data/net/minecraft/world/entity/Entity.mapping

```diff
@@ -142,6 +149,8 @@ CLASS net/minecraft/world/entity/Entity
 		ARG 1 livingEntity
 	METHOD getEncodeId ()Ljava/lang/String;
 		COMMENT Returns the string that identifies this Entity's class
+	METHOD getExitPortal (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/core/BlockPos;Z)Ljava/util/Optional;
+		ARG 3 isToNether
```

Comment:
Could add mappings for the `level` and `pos` params.

Reply:
Done, but it seems like I missed some, ones that are red in enigma should be ones that I missed

Reply:
I ignored one that are not "float1" or "integer1" but sometimes named parameters had those names. I should recheck every parameter